entity-server-api/src/main/java/org/terracotta/entity/ConcurrencyStrategy.java

```diff
@@ -38,20 +41,22 @@
   public final int UNIVERSAL_KEY = Integer.MIN_VALUE; 
   /**
    * MANAGEMENT_KEY is a zero key which is used to manage the entity (anything other than invoke action).
+   * Additionally, a message can choose to use this key if it must be run exclusively within the entity.  Be careful with
+   * the cases where this is done, however, since the MANAGEMENT_KEY cannot be used for synchronizing data.
```

Comment:
I wonder if MANAGEMENT_KEY actually has much meaning for an entity implementation.  For the platform, it's used for operations that are fully exclusive for an entity like lifecycle operations where no other operations can be running on any concurrency key.  If an entity needs to have 100% serial execution it makes more sense for that entity to define one concurrency > 0 and use that key exclusively.  Therefore, I'm having a hard time coming up with a scenario where an entity should actually use the MANAGEMENT_KEY in an invoke.